extensions/amp-bind/0.1/bind-expression.js

```diff
@@ -378,6 +386,27 @@ export class BindExpression {
             ? this.eval_(args[1], scope)
             : this.eval_(args[2], scope);
 
+      case AstNodeType.ARROW_FUNCTION:
+        const functionScope = map(scope);
+        return (...values) => {
+          // Support parameters in arrow functions by forwarding their values
+          // into the function's scope. For example, in this function call:
+          //
+          //     const f = (x, y) => x + y;
+          //     f(2, 7);
+          //
+          // `names` == ['x', 'y'] and `values` == [2, 7], so we include
+          // {x: 2, y: 7} in the scope when evaluating `x + y`.
+
+          const names = this.eval_(args[0], scope);
```

Comment:
This is just to extract `args[0]` is guaranteed to b an array of Literals, right? If so, can we just extract `arg[0].value` instead of "evaling" the literals?

Reply:
Good observation. I actually had something similar originally but decided it wasn't worth the cost of readability. This PR already adds a special case for arrow functions above.